community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/log/entry/LogEntryWriter.java

```diff
@@ -73,6 +85,13 @@ public void serialize( TransactionRepresentation tx ) throws IOException
         tx.accept( serializer );
     }
 
+    public void serialize( CommittedTransactionRepresentation tx ) throws IOException
```

Comment:
I don't think it is worth adding those three new methods here "only" for the test. I would rather have them as helper functions in the test.